staging/src/kubevirt.io/client-go/kubecli/vmi.go

```diff
@@ -152,6 +152,7 @@ func RequestFromConfig(config *rest.Config, resource, name, namespace, subresour
 		u.Path,
 		fmt.Sprintf("/apis/subresources.kubevirt.io/%s/namespaces/%s/%s/%s/%s", v1.ApiStorageVersion, namespace, resource, name, subresource),
 	)
+	u.RawQuery = strings.Join(params, "&")
```

Comment:
I can see that this will work as expected with the port paramter, but I am not sure this is in general safe (like when it gets extended). Can we instead e.g. pass key/value pairs and properly encode the query string with `url.Values{}`?

Does not have to  be exactly that solution, but something more safe.

Reply:
Done